src/command_line_parser.h

```diff
@@ -99,10 +99,10 @@ struct Option {
     return lo;
   }
 
-  const int id_;
-  const std::string flag_;
-  const std::string arg_desc_;
-  const std::string desc_;
+  int id_;
+  std::string flag_;
+  std::string arg_desc_;
+  std::string desc_;
```

Comment:
Why dropping const flag?

Reply:
Complaints from implicit copies on creating the vectors. Might be able to specify `Option&` to fix that though, let me check.